torch/_inductor/fx_passes/reinplace.py

```diff
@@ -496,23 +496,42 @@ def can_inplace(node, mutated_arg):
             )
 
     def log_inplace_results(
-        node_name,
-        old_tensors_to_clone,
-        tensors_to_clone,
-        possibly_missed_reinplacing_opportunities,
+        node_name, old_tensors_to_clone, tensors_to_clone, missed_args, missed_nodes
     ):
+        # Total size of possibly_missed_reinplacing_opportunities for tensors with static shapes.
+        missed_bytes = 0
+
+        def bytes(node):
+            if node.meta.get("val") is not None:
+                t = node.meta["val"]
+                return t.element_size() * t.numel()
+            else:
+                return 0
```

Comment:
what does this do when it hits a symint?

Reply:
the new check is          
   if t is not None and isinstance(t.element_size(), int) and isinstance(t.numel(), int):